src/test/java/ui/ProductAddTest.java

```diff
@@ -0,0 +1,35 @@
+package ui;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import uiPages.LoginPage;
+import uiPages.ProductPage;
+
+import static helpers.Utilities.getPropertyFromAppProp;
+
+public class ProductAddTest extends BaseTest {
+
+    private ProductPage productPage = null;
+
+    private String specificProduct;
+
+    public ProductAddTest() {
+        this.specificProduct = getPropertyFromAppProp("product1");
+    }
+
+    @BeforeMethod
+    void beforeMethod() {
+        this.productPage = new ProductPage(getDriver());
+    }
+
+    @Test
+    public void addProductToCart() {
```

Comment:
This test should be implemented so it can be run for any product. Products have different color or sizes available. So the test should take as input beside the product URL, the size, the color and the product name.

Reply:
I will address this in a future branch.

Reply:
Sounds good. I added ticket #50 for this and assign it to you.